lib/pubnub/client/getters_setters.rb

```diff
@@ -49,12 +47,16 @@ def change_uuid(uuid)
       # ==============
       # Can't change user_id while subscribed. You have to leave every subscribed channel.
       def change_user_id(user_id)
-        change_uuid(user_id)
+        Pubnub.logger.debug('Pubnub::Client') { 'Changing user_id' }
+        raise('Cannot change user_id while subscribed.') if subscribed?
+        Validator::Client.validate_user_id user_id
+
+        @env[:uuid] = user_id
```

Comment:
@kleewho why not `@env[:user_id] = user_id`?
`env` hash used internally, lets change names now, so it future we won't think why only some of them has been changed.

Reply:
I wasn't sure if it's used or not by our users.